plugin/src/helpers/edge.ts

```diff
@@ -30,45 +32,42 @@ const loadMiddlewareManifest = (netlifyConfig: NetlifyConfig): Promise<Middlewar
 /**
  * Convert the Next middleware name into a valid Edge Function name
  */
-const sanitizeName = (name: string) => `next${name === '/' ? '_index' : name.replace(/\W/g, '_')}`
+const sanitizeName = (name: string) => `next_${name.replace(/\W/g, '_')}`
 
 /**
  * Initialization added to the top of the edge function bundle
  */
 const bootstrap = /* js */ `
+globalThis.process = { env: {...Deno.env.toObject(), NEXT_RUNTIME: 'edge', 'NEXT_PRIVATE_MINIMAL_MODE': '1' } }
 globalThis._ENTRIES ||= {}
+// Deno defines "window", but naughty libraries think this means it's a browser
```

Comment:
Anecdotal, but it seems odd they define the `window` object for something that never runs in the browser, or is this maybe for some stuff related to WASM?